libs/application/template-api-modules/src/lib/modules/templates/social-insurance-administration/social-insurance-administration-utils.ts

```diff
@@ -463,6 +464,23 @@ export const getIncomeTypes = (application: Application): IncomeTypes[] => {
   }))
 }
 
+export const shouldDistributeIncomeByMonth = (
+  application: Application,
+): boolean => {
+  // Let TR know if there is any case where income is uneven during the year
+  const { incomePlan } = getIPApplicationAnswers(application.answers)
+
+  if (
+    incomePlan.find(
+      (i) =>
+        i?.unevenIncomePerYear?.[0] === YES && i?.incomeCategory === INCOME,
+    )
+  ) {
+    return true
+  }
+  return false
+}
```

Comment:
Very small nitpick but since you only want a boolean value and not the actual value (income plan) then it would be more appropriate to use `.some` instead of `.find`

also can shorten the amount of lines
```suggestion
 const hasUnevenIncome = incomePlan.some(
    (i) =>
      i?.unevenIncomePerYear?.[0] === YES && i?.incomeCategory === INCOME,
  )
  return hasUnevenIncome
```

Reply:
Cool, thanks ❇️